app/model/SubscriptionData.scala

```diff
@@ -54,7 +54,10 @@ case class SubscriptionData(
    personalData: PersonalData,
    paymentData: PaymentData,
    promoCode: Option[PromoCode],
-   currency: Currency
+   currency: Currency,
+   ophanPageViewId: Option[String],
+   ophanVisitId: Option[String],
+   ophanBrowserId: Option[String]
```

Comment:
I notice all the person/payment data is grouped into case classes, maybe it's worth having an OphanData case class to keep this neat?